nrf52-code/usb-lib-solutions/complete/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+authors = ["Ferrous Systems"]
+edition = "2018"
```

Comment:
Nit: Shall we raise to 2021 at some point?

Reply:
#117